### DISCUSSION TITLE
GLTF model separation with vertex groups and triggering individual scenario when pressed

### DISCUSSION THREAD

**Discussion by Whiteblodd** [google/model-viewer#4133] [Q&A]
Hi! I'm developing an anatomical human model, and I want to make a walk thru the individual body parts with changing textures, camera views, and scene center point, with the ability to move thru the levels of interaction. 

I am wondering how I can implement my body parts separation on vertex groups as user interaction to dive into the model. 
Is there any way to take this data from blender and import it into the viewer as instances?
How I can identify a user clicking on a certain part of the body?

**Comment by elalish** [comment]
Our API is material-based, so I'd recommend authoring your file in such a way that each region you want independently-selectable has its own glTF material. Then you can use [materialFromPoint](https://modelviewer.dev/docs/index.html#entrydocs-scenegraph-methods-materialFromPoint) to identify the selected region. See https://modelviewer.dev/examples/scenegraph/#pickMaterialExample

**Reply by elalish** [reply]
Also, please share a show & tell when you get this working - it sounds really interesting and we'd love to see the results!
